quiz/backend/players/serializers.py

```diff
@@ -10,7 +11,33 @@ class PlayerSerializer(ModelSerializer):
     :rtype: rest_framework.serializers.Serializer
     """
 
+    username = CharField(source="nick", required=True, validators=[validate_profanity])
+
+    class Meta:
+        model = models.Player
+        fields = (
+            "uuid",
+            "username",
+        )
+        read_only_fields = ("uuid",)
+
+
+class PlayerDetailSerializer(ModelSerializer):
+    """Serializer for Player model.
+
+    :returns: PlayerSerializer
+    :rtype: rest_framework.serializers.Serializer
+    """
+
+    username = CharField(source="nick", validators=[validate_profanity])
+
     class Meta:
         model = models.Player
-        fields = ("uuid", "nick", "rank", "created_at")
-        read_only_fields = ("uuid", "rank", "created_at")
+        fields = (
+            "uuid",
+            "username",
+            "rank",
+            "created_at",
+        )
+        read_only_fields = ("uuid", "created_at")
```

Comment:
Shouldn't `rank` also be a read only field or do we want players to update their rank by hand?